app/controllers/open_library_books_controller.rb

```diff
@@ -0,0 +1,11 @@
+class OpenLibraryBooksController < ApplicationController
+  def book_information
+    response = OpenLibraryBooks.new(params[:isbn]).retrieve_book_information
```

Comment:
El ISBN es un parametro requerido, sin ese parametro vamos a tener problemas con el service :eyes: en este caso es conveniente que usemos [strong_params](https://api.rubyonrails.org/classes/ActionController/Parameters.html) y atender ese potencial conflicto de antemano
```suggestion
    response = OpenLibraryBooks.new.retrieve_book_information(isbn_param)
    
    private
    
    def isbn_param
      params.require(:isbn)
    end
```